conf/openmetadata.yaml

```diff
@@ -199,6 +199,7 @@ elasticsearch:
 eventMonitoringConfiguration:
   eventMonitor: ${EVENT_MONITOR:-prometheus}  # Possible values are "prometheus", "cloudwatch"
   batchSize: ${EVENT_MONITOR_BATCH_SIZE:-10}
+  pathPattern: ${EVENT_MONITOR_PATH_PATTERN:-["/*"]}
```

Comment:
we should also what metrics we want to monitor, like P90 , p95, p99 etc.. cc @ulixius9

Reply:
Okay, will have a look into it.

Reply:
also lets add few endPoints we want to monitor
/api/v1/tables
/api/v1/healthcheck
cc @mohityadav766

Reply:
Sure, thank you for the feedback.

Reply:
Is it possible to also have an exclusion pattern?
Ex: EVENT_MONITOR_EXCLUDE_PATH_PATTERN